Fixes (#18552)
blakeblackshear/frigate: docs/docs/configuration/authentication.md

    - 172.18.0.0/16 # <---- this is the subnet for the internal Docker Compose network
```

## Session Length

> The default session length for user authentication in Frigate is 24 hours. This setting determines how long a user's authenticated session remains active before a token refresh is require, otherwise another login will be required.

Author: ```suggestion
The default session length for user authentication in Frigate is 24 hours. This setting determines how long a user's authenticated session remains active before a token refresh is required — otherwise, the user will need to log in again.
```

While the default provides a balance of security and convenience, you can customize this duration to suit your specific security requirements and user experience preferences. The session length is configured in seconds.

> The default value of `86400` will expire the authentication session after 24 hours, some other examples:

Author: ```suggestion
The default value of `86400` will expire the authentication session after 24 hours. Some other examples:
```
